### PR TITLE
Enable remaining Windows suppressed tests

### PR DESCRIPTION
**Pull request by alan-knight**

### CODE REVIEW

File: dwds/test/chrome_proxy_service_test.dart [hunk 1]
```diff
@@ -126,12 +126,15 @@ void main() {
...
           false: true,
         };
+
         var result =
             await service.callServiceExtension(serviceMethod, args: args);
         expect(
             result.json,
             args.map((k, v) => MapEntry(k is String ? k : jsonEncode(k),
                 v is String ? v : jsonEncode(v))));
+      }, onPlatform: {
```
Comment (jakemac53) [inline dwds/test/chrome_proxy_service_test.dart]: Why do we need an extra skip here? was this flaky?

File: dwds/test/chrome_proxy_service_test.dart [hunk 2]
```diff
@@ -485,6 +488,13 @@ void main() {
       });
 
       test('returns null if not paused', () async {
+        // In case we haven't started yet, try to resume.
+        try {
+          await service.resume(isolateId);
```
Comment (grouma) [inline dwds/test/chrome_proxy_service_test.dart]: Why is this necessary? Is this related to https://github.com/dart-lang/webdev/issues/716?

File: dwds/test/chrome_proxy_service_test.dart (hunk 1 above)
Comment (alan-knight) [reply dwds/test/chrome_proxy_service_test.dart]: Yes, it's very flaky, and when it flakes it times out and causes numerous other tests to fail.

File: dwds/test/chrome_proxy_service_test.dart (hunk 2 above)
Comment (alan-knight) [reply dwds/test/chrome_proxy_service_test.dart]: It looks like it. This is one of the flakiest test, and when it fails it's because it gets the single-entry stack that is returned when paused on start rather than the null when we're not paused.

Comment (grouma) [reply dwds/test/chrome_proxy_service_test.dart]: Why are we paused at the start here? Can we simply wait for the application to be running?

Comment (alan-knight) [reply dwds/test/chrome_proxy_service_test.dart]: What do we wait for?

**Comment by alan-knight** [comment]
There are still some possible issues, but the Windows tests are so flaky I'm going to merge this and we can fix up later.
